addons/edi_notifier/models/email_notifier.py

```diff
@@ -99,10 +99,12 @@ def get_email_model(self):
     _timestamp_field = "x_last_checked_not_received"
 
     def _get_time_today(self, cron):
-        time = fields.Datetime.from_string(cron.nextcall).time()
-        return datetime.now().replace(
-            hour=time.hour, minute=time.minute, second=0, microsecond=0,
-        )
+        time = fields.Datetime.from_string(cron.nextcall)
+        time_now = datetime.now()
+        # if the date is not today's date then return the next call date
```

Comment:
Suggestion (feel free to resolve): This comment would probably be better as a doc-string :)